tests/test_upload_sync.py

```
@@ -0,0 +1,376 @@
+
+# ===========================================================================
+# TEST: BLB Protection (CRITICAL)
+# ===========================================================================
+
+class TestBLBProtection:
+    """BLB tables must NEVER be dropped by sync_schema."""
+
+    def test_blb_tables_never_in_drop_list(self, mgr):
+        """sync_schema must not include BLB tables in DROP statements."""
+        tables_to_sync = mgr._get_tables()
+        blb_in_list = [t for t in tables_to_sync if t in mgr.BLB_TABLES]
```

Ruff (0.15.2)

[error] Local variable `blb_in_list` is assigned to but never used

Remove assignment to unused variable `blb_in_list`

(F841)